modules/packages/Curl.chpl

```diff
@@ -547,9 +549,10 @@ module Curl {
         length = this.length;
         return 0;
       }
-      override proc getpath(out path:c_string, out len:int(64)):errorCode {
-        path = qio_strdup(this.url_c);
-        len = url_c.size;
+      override proc getpath(out path:c_ptr(uint(8)), out len:int(64)):errorCode {
+        use OS.POSIX;
```

Comment:
I'd rather see it call an unstable string-length-for-ptr helper in CTypes than to `use POSIX`. This applies elsewhere as well.

Reply:
updated this similar to https://github.com/chapel-lang/chapel/pull/22622#discussion_r1283606057